Add note about samples from the same genus in normalized compendia (#120)
AlexsLemonade/refinebio-docs: docs/main_text.md

We've made our analyses underlying processing choices and exploring test compendia available at our <a href = "https://github.com/AlexsLemonade/compendium-processing" target = "blank">`compendium-processing`</a> repository.

### Collapsing by genus

Microarray platforms are generally designed to assay samples from a specific species.

> In some cases, publicly available data surveyed by refine.bio may include samples run on a microarray platform that was not specifically designed for that species (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 

Reviewer: ```suggestion
In some cases, publicly available data surveyed by refine.bio may include samples where the microarray platform used was not specifically designed for the species as described (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 
```

> In some cases, samples that are not from the species for which a microarray platform is specifically designed for are in publicly available data surveyed by refine.bio (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 

Reviewer: ```suggestion
In some cases, publicly available data surveyed by refine.bio may include samples run on a microarray platform that was not specifically designed for that species (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 
```

> When we encounter this in refine.bio, we will include samples in a compendium from species that differ from the primary platform species when the two species share a genus (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium).

Reviewer: ```suggestion
When we encounter this in refine.bio, we will include samples in a compendium from species that differ from the primary platform species when the two species share a genus (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium, and _Mus_ crosses are included in the _Mus musculus_ normalized compendium).
```

Keep it parallel?

> When we encounter this in refine.bio, we will include samples from the species that differs from the platform species when the two species share a genus in the larger species' normalized compendium (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium).

Reviewer: ```suggestion
When we encounter this in refine.bio, we will include samples in a compendium from species that differ from the primary platform species when the two species share a genus (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium).
```

My change loses the directionality of the the larger species, but I replaced that with "primary" which I hope is clear enough?

> This is generally a low number of samples compared to the total number of samples included in a normalized compendium.

Reviewer: ```suggestion
Such non-primary species are generally a small number of samples compared to the total number of samples included in a normalized compendium.
```
Trying to clarify the hanging "this"

Author: What do you think about the change in https://github.com/AlexsLemonade/refinebio-docs/pull/120/commits/740e1c8a3537577b1241fdbc1ad0262f0c27f258? I'm not sure it's quite there yet.

> If you would like to filter a normalized compendium based on a sample's species label, you can utilize the `refinebio_organism` column in the metadata TSV file or the `.samples[].refinebio_organism` field in the metadata JSON file included as part of the download.

Reviewer: ```suggestion
If you would like to filter a normalized compendium based on a sample's species label, you can use the `refinebio_organism` column in the metadata TSV file or the `.samples[].refinebio_organism` field in the metadata JSON file included as part of the download.
```

I hate utilize most of the time. It might be okay here, so feel free to ignore my persnickitiness.

We've made our analyses underlying processing choices and exploring test compendia available at our <a href = "https://github.com/AlexsLemonade/compendium-processing" target = "blank">`compendium-processing`</a> repository.

### Collapsing by genus

Microarray platforms are generally designed to assay samples from a specific species.

> In some cases, publicly available data surveyed by refine.bio may include samples where the microarray platform used was not specifically designed for the species as described (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 

Reviewer 2: Should we say `were run` instead of `will be run` to make it more obvious that these runs were already done by external researchers?

Author: How about `are run` ?

When we encounter this in refine.bio, we will include samples in a compendium from species that differ from the primary platform species when the two species share a genus (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium, and _Mus_ crosses are included in the _Mus musculus_ normalized compendium).

> Such non-primary species samples generally account for a low number of the total samples included in a normalized compendium.

Reviewer: ```suggestion
Such non-primary species samples generally account for a small fraction of the total samples included in a normalized compendium.
```
